build-tools/packages/build-tools/src/common/monoRepo.ts

```diff
@@ -86,9 +89,15 @@ export class MonoRepo {
         this.version = "";
         const pnpmWorkspace = path.join(repoPath, "pnpm-workspace.yaml");
```

Comment:
Nit: the other variables use "fooPath" rather than "fooWorkspace" - should we use a consistent naming scheme for all of them?

Reply:
Clarified the variable names.